pkg/rest/retry.go

```diff
@@ -60,7 +60,7 @@ func GetWithRetry(client *http.Client, url string, settings RetrySetting) (resp
 	}
 
 	for i := 0; i < settings.MaxRetries; i++ {
-		log.Warn("Retrying failed GET request %s with error (HTTP %d)", url, resp.StatusCode)
+		log.Warn("Failed to get config %s with error (HTTP %d). Waiting for %s before retrying...", url, resp.StatusCode, settings.WaitTime)
```

Comment:
nit: These are not logging the `config` but the `url` - I think logging 

> Failed to get config https://my-end.live.dynatrace.com/api/something/something/uuid ..

reads a bit strange. 

Note that the Send one you're attempting to unify with, actually logs the `objectName`- maybe that's possible here as well. Maybe both the name and API call could be helpful also